pulp_rpm/app/viewsets.py

```diff
@@ -348,39 +322,41 @@ class UlnRemoteViewSet(RemoteViewSet):
                 "action": ["retrieve"],
                 "principal": "authenticated",
                 "effect": "allow",
-                "condition": "has_model_or_obj_perms:rpm.view_ulnremote",
+                "condition": "has_model_perms:rpm.view_ulnremote",
```

Comment:
Actually this should be model or object, since retrieve is a detail call to an existing object.
Create and list are the non-detail calls usually.